rotkehlchen/tests/unit/decoders/test_ens.py

```diff
@@ -80,3 +81,210 @@ def test_mint_ens_name(database, ethereum_manager, function_scope_messages_aggre
+            balance=Balance(),
+            location_label='0x4bBa290826C253BD854121346c370a9886d1bC26',
+            notes='Set ENS com.github for nebolax.eth',
+            counterparty='ens',
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=HexBytes(
+                '0xaa59cb2029651d2ed2c0d1ee34b9b88f0b90278fc6da5b51446d4abf24d7f598',
+            ),
+            sequence_index=295,
+            timestamp=1664548859000,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(),
+            location_label='0x4bBa290826C253BD854121346c370a9886d1bC26',
+            notes='Set ENS com.reddit for nebolax.eth',
+            counterparty='ens',
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=HexBytes(
+                '0xaa59cb2029651d2ed2c0d1ee34b9b88f0b90278fc6da5b51446d4abf24d7f598',
+            ),
+            sequence_index=296,
+            timestamp=1664548859000,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(),
+            location_label='0x4bBa290826C253BD854121346c370a9886d1bC26',
+            notes='Set ENS com.twitter for nebolax.eth',
+            counterparty='ens',
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=HexBytes(
+                '0xaa59cb2029651d2ed2c0d1ee34b9b88f0b90278fc6da5b51446d4abf24d7f598',
+            ),
+            sequence_index=297,
+            timestamp=1664548859000,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(),
+            location_label='0x4bBa290826C253BD854121346c370a9886d1bC26',
+            notes='Set ENS org.telegram for nebolax.eth',
+            counterparty='ens',
+            identifier=None,
+            extra_data=None,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0x4bBa290826C253BD854121346c370a9886d1bC26']])  # noqa: E501
+def test_set_resolver(evm_transaction_decoder):
+    """
+    Data taken from
+    https://etherscan.io/tx/0xae2cd848ce02c425bc50a8f46f8430eec32234475efb6fcff28315d2791329f6
+    """
+    # TODO: For faster tests hard-code the transaction and the logs here so no remote query needed
+    tx_hash = deserialize_evm_tx_hash('0xae2cd848ce02c425bc50a8f46f8430eec32234475efb6fcff28315d2791329f6')  # noqa: E501
+    events = evm_transaction_decoder.decode_transaction_hashes(ignore_cache=True, tx_hashes=[tx_hash])  # noqa: E501
```

Comment:
Again please, for developing it might be faster but once you have finished is not much work to hardcode the transaction